added_resistors_and_capasitors

vytzal opened the pull request.

vytzal committed:
added_resistors_and_capasitors

arturas-aleksandrovas commented:
Pakeitimai nėra priimti. Bibliotekoje nereikia kurti naujų bylų, tereikia modifikuoti esamas - įvedant naujus komponentus. Komponento pavadinimas = gamintojo kodas, kuriant naujus komponentus nereikia perrašyti jau esamų, reikia copy-paste ir tuomet keisti. Platesnė informacija Moodle sistemoje Komponentų bibliotekos.pdf

vytzal committed:
added

resistors:
ERJ-UP3J471V
ERA-6VEB4700V
RN73R2BTTD4700D25
capacitors:
VJ0603D0R1CXBAJ
VJ0805D0R1BXPAJHT
12062A0R5CAT2A

vytzal committed:
added

resistors:
ERJ-UP3J471V
ERA-6VEB4700V
RN73R2BTTD4700D25
capacitors:
VJ0603D0R1CXBAJ
VJ0805D0R1BXPAJHT
12062A0R5CAT2A

Aleksandrovas committed:
Merge branch 'KTU_Lib_master' into pr/32

Aleksandrovas commented:
Varžų aprašą (description) reikėtų išlaikyti panašu į kitų komponentų:
RES 470R, ±5%, 250mW, 150V, 0603 [1608 Metric], Thick Film
CAP MLCC, 0.5uF, 200V, 0805 [2012 Metric], ±0.25pF, C0G (NP0)

Komponentų footprint nereikėjo keisti, juk užtenka pasirinkti reikiamą šabloną pagal dydį ir jį koreguoti.
Trūksta atidumo kondensatorių value nesutampa.

Pataisyta, tačiau reikia atidžiau įvedinėti komponentus.
